index.html

```diff
@@ -54,5 +55,148 @@ <h1>I’m Fadahunsi Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <main>
+        <section id="myWork" class="my-works">
+            <div class="work-item">
+                <img alt="work-item-img" src="./img/work/work-1-cover.png"/>
+              <h3 class="work-title">Tonic</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
```

Comment:
- [ ] it would be better if you use the right font in project typography (title, button, paragraph.....etc) so you should use 
![image](https://user-images.githubusercontent.com/61790587/170544191-2bd3150a-f2a6-459f-a0c9-dcb0dbe20405.png)